src/System.Private.DataContractSerialization/src/System/Xml/ValueHandle.cs

```diff
@@ -88,15 +88,8 @@ public ValueHandle(XmlBufferReader bufferReader)
             _type = ValueHandleType.Empty;
         }
 
-        private static Base64Encoding Base64Encoding
-        {
-            get
-            {
-                if (s_base64Encoding == null)
-                    s_base64Encoding = new Base64Encoding();
-                return s_base64Encoding;
-            }
-        }
+        private static Base64Encoding Base64Encoding => s_base64Encoding ??= new Base64Encoding();
```

Comment:
How does this reduce allocations?

Reply:
Sorry, it doesn't, just saw this while working in the file and thought might as well refactor.
Can remove if it's too 'off-topic'